bliss/conf/base_config.yaml

```diff
@@ -81,8 +81,10 @@ encoder:
     input_transform_params:
         use_deconv_channel: false
         concat_psf_params: false
-        log_transform: true
+        log_transform: false
+        rolling_z_score: true
         z_score: false
+        clahe: false
```

Comment:
Can we remove clahe and z_score, if rolling z score is all we need? (I'd like to keep our software simple to use by minimizing the number of configuration options, even at the expense of some flexibility.)

Reply:
Would it be worth keeping the `z_score` config option around for the multiband paper? I'm assuming we'll want to talk about the global z-scoring procedure a bit.

Reply:
Either way. I wouldn't really call the z_scoring procedure global though -- it just operates on 80x80 the subimages that we use for training.

Reply:
The way that `z_scoring` procedure works though within the `transforms.py` code is that it takes the mean and std over the entire image, not just 80x80 patches. On real data it computes those values over the whole image.

Reply:
That's more a defect though than a feature. It makes it so the training data the prediction data are from a different in distribution.

Reply:
Thus my lack of enthusiasm for the class z_scoring procedure.

Reply:
So if we have to re-implement that z_scoring procedure, we should probably just get rid of it here then.